galeracluster/source/documentation/audit-log-plugin.rst

```diff
@@ -0,0 +1,821 @@
+.. meta::
+   :title: Audit Log Plugin
+   :description: Galera Audit Log Plugin monitors and logs connection and query activity performed on a specific server.
+   :language: en-US
+   :keywords: galera cluster, audit log
+   :copyright: Codership Oy, 2014 - 2021. All Rights Reserved.
+
+.. container:: left-margin
+
+   .. container:: left-margin-top
+
+      :doc:`The Library <../index>`
+
+   .. container:: left-margin-content
+
+      .. cssclass:: here
+
+         - :doc:`Documentation <./index>`
+
+      - :doc:`Knowledge Base <../kb/index>`
+      - :doc:`Training <../training/index>`
+
+      .. cssclass:: sub-links
+
+         - :doc:`Tutorial Articles <../training/tutorials/index>`
+         - :doc:`Training Videos <../training/videos/index>`
+
+      - :doc:`FAQ <../faq>`
+      - :ref:`search`
+
+      Related Documents
+
+      - :doc:`log`
+      - :doc:`galera-parameters`
+
+.. container:: top-links
+
+   - `Home <https://galeracluster.com>`_
+
+   .. cssclass:: here
+
+      - :doc:`Docs <./index>`
+
+   - :doc:`KB <../kb/index>`
+
+   .. cssclass:: nav-wider
+
+      - :doc:`Training <../training/index>`
+
+   - :doc:`FAQ <../faq>`
+
+
+.. cssclass:: library-document
+.. _`auditlogplugin`:
+
+=================
+ Audit Log Plugin
+=================
+
+.. index::
+   pair: Descriptions; Audit Log Plugin
+.. index::
+   pair: Logs; Audit Log
+
+Galera Audit Log Plugin monitors and logs connection and query activity performed on a specific server. Information on the activity is stored in the XML log file, where each event will have a ``NAME`` field, a unique ``RECORD_ID`` field and a ``TIMESTAMP`` field. This implementation is alternative to the MySQL Enterprise Audit Log Plugin.
```

Comment:
The Galera Audit Log Plugin...

Reply:
I'm not even sure if we should call it the Galera Audit Log Plugin. I think "The Audit Log Plugin" will suffice

Reply:
Fixed and pushed.